### PR TITLE
Fix West Virginia low-income family tax credit

### LINKED ISSUE #3176: Fix West Virginia low-income family tax credit
In many cases, the 0.507.1 version produces incorrect amounts for the `wv_low_income_family_tax_credit` variable.   Below is a test case that fails because PEUS mistakenly assigns this high-income family a credit of about $2,000.
```
- name: Tax unit with taxsimid 82215 in e21.its.csv and e21.ots.csv
  absolute_error_margin: 0.01
  period: 2021
  input:
    people:
      person1:
        age: 62
        employment_income: 160_010
        taxable_interest_income: 5_505
        ssi: 0  # not in TAXSIM35
        state_supplement: 0  # not in TAXSIM35
        wic: 0  # not in TAXSIM35
      person2:
        age: 62
        employment_income: 158_010
        taxable_interest_income: 5_505
        ssi: 0  # not in TAXSIM35
        state_supplement: 0  # not in TAXSIM35
        wic: 0  # not in TAXSIM35
    tax_units:
      tax_unit:
        members: [person1, person2]
        premium_tax_credit: 0  # not in TAXSIM35
        local_income_tax: 0  # not in TAXSIM35
        state_sales_tax: 0  # not in TAXSIM35
    spm_units:
      spm_unit:
        members: [person1, person2]
        snap: 0  # not in TAXSIM35
        tanf: 0  # not in TAXSIM35
    households:
      household:
        members: [person1, person2]
        state_fips: 54  # WV
  output:  # expected results from patched TAXSIM35 2023-10-05 version
    wv_income_tax_before_non_refundable_credits: 20_001.95
    wv_low_income_family_tax_credit: 0  # PEUS 0.507.1 produces 2_000.20
    wv_income_tax: 20_001.95  # PEUS 0.507.1 produces 18_001.75
```

### PR DESCRIPTION
**Pull request by yedeer12**
Fixes #3176

### Summary
:memo::white_check_mark::hammer_and_wrench:

This pull request refactors and fixes the codebase and the parameters for the West Virginia Low Income Family Tax Credit (LIFTC). It renames the LIFTC parameters from `low_income_family_tax_credit` to `liftc`, adds comments and print statements, and corrects the bracket logic for the LIFTC amount. It also adds a changelog entry file and new test cases to verify the model behavior for the LIFTC.

> _`LIFTC` refactored_
> _Tests and changelog added_
> _Spring cleaning the code_

### Walkthrough
*  Rename LIFTC parameters and files from `low_income_family_tax_credit` to `liftc` for brevity and consistency ([link](https://github.com/PolicyEngine/policyengine-us/pull/3195/files?diff=unified&w=0#diff-36649e7565f6bcb8bada6f03b92002a4f420bb625310bcda22f41beada44380e), [link](https://github.com/PolicyEngine/policyengine-us/pull/3195/files?diff=unified&w=0#diff-fd55ed19360e8b759853abb636d0dbaddf1339aec76b2d09503f8aae04191842), [link](https://github.com/PolicyEngine/policyengine-us/pull/3195/files?diff=unified&w=0#diff-53008273872df8989e06accbf7f2b4535569adaec2c6b54d3630af0a8a2d08b1), [link](https://github.com/PolicyEngine/policyengine-us/pull/3195/files?diff=unified&w=0#diff-a7bc1a6371a223e60f6359d216059237af2e85b5499042d6e1c14ab2be188e49), [link](https://github.com/PolicyEngine/policyengine-us/pull/3195/files?diff=unified&w=0#diff-d54867374dab713d625e6741f794af6c5eb6f47ccacbd1a374cabe54c942e907), [link](https://github.com/PolicyEngine/policyengine-us/pull/3195/files?diff=unified&w=0#diff-242a1fb5abd6d49504fabcacda2cc20d3cffa1200e2778e51000ca6aa9aea335), [link](https://github.com/PolicyEngine/policyengine-us/pull/3195/files?diff=unified&w=0#diff-eaa80cdff49897b2c5295c74b926fb00172e716c86790532d1b1110e00ae819d), [link](https://github.com/PolicyEngine/policyengine-us/pull/3195/files?diff=unified&w=0#diff-26a60bb34bb7c3a21eab95e4ab652a2e764f5a03bf5ba87e3d0f9ed242fd7988L15-R15), [link](https://github.com/PolicyEngine/policyengine-us/pull/3195/files?diff=unified&w=0#diff-f5a1539b891ed5a3931c7b301b7d05908aaabacb57e88ac49aa3cdbb63d5b0a9L17-R17))
*  Fix bug in LIFTC amount calculation by using right endpoint of brackets instead of left endpoint ([link](https://github.com/PolicyEngine/policyengine-us/pull/3195/files?diff=unified&w=0#diff-f5a1539b891ed5a3931c7b301b7d05908aaabacb57e88ac49aa3cdbb63d5b0a9L34-R39))
*  Add comment to explain first bracket condition for LIFTC amount ([link](https://github.com/PolicyEngine/policyengine-us/pull/3195/files?diff=unified&w=0#diff-f5a1539b891ed5a3931c7b301b7d05908aaabacb57e88ac49aa3cdbb63d5b0a9R23))
*  Add print statements to debug LIFTC amount and credit percentage ([link](https://github.com/PolicyEngine/policyengine-us/pull/3195/files?diff=unified&w=0#diff-f5a1539b891ed5a3931c7b301b7d05908aaabacb57e88ac49aa3cdbb63d5b0a9L45-R47))
*  Add integration test case for high-income tax unit with zero LIFTC ([link](https://github.com/PolicyEngine/policyengine-us/pull/3195/files?diff=unified&w=0#diff-5b2ddee3ea35af1ef3a0424aa101b604008ea9df0a1efab4596b8c44c34bc002R1-R38))
*  Add unit test case for single filer with income above federal poverty guidelines and zero LIFTC ([link](https://github.com/PolicyEngine/policyengine-us/pull/3195/files?diff=unified&w=0#diff-6caf7ed425eec388c3b39a06bd5f3c11326f0fbc1ccaa309ba13bf5b886b75baR1-R14))
*  Delete unused test file for LIFTC variable ([link](https://github.com/PolicyEngine/policyengine-us/pull/3195/files?diff=unified&w=0#diff-5b05741b6382f7b25ef24bc97f4826dd84d477b0fa25a5fceb275d960af6dbcc))
*  Rename test file for LIFTC federal poverty guidelines variable ([link](https://github.com/PolicyEngine/policyengine-us/pull/3195/files?diff=unified&w=0#diff-a8800bbfbafa0f7a8e8773719cf81ed2ca9714917c68d0ea348fd50d012b2bdd))
*  Add empty YAML file for changelog entries ([link](https://github.com/PolicyEngine/policyengine-us/pull/3195/files?diff=unified&w=0#diff-5885042762833a4aef51405a4036067e4e6aefa7a5091a2987d1d9d1fc2647e8R1))

### CODE REVIEW

File: policyengine_us/tests/policy/baseline/gov/states/wv/tax/income/credits/liftc/intergration.yaml
Comment (MaxGhenis) [inline policyengine_us/tests/policy/baseline/gov/states/wv/tax/income/credits/liftc/intergration.yaml:1]: rename `integration.yaml`

**Review by MaxGhenis — CHANGES_REQUESTED** [review]

File: policyengine_us/variables/gov/states/wv/tax/income/credits/liftc/wv_low_income_family_tax_credit.py [hunk 1]
```diff
@@ -31,16 +32,17 @@ def formula(tax_unit, period, parameters):
                 filing_status == filing_statuses.WIDOW,
             ],
             [
-                p.amount.single.calc(reduced_agi),
-                p.amount.separate.calc(reduced_agi),
-                p.amount.joint.calc(reduced_agi),
-                p.amount.head_of_household.calc(reduced_agi),
-                p.amount.widow.calc(reduced_agi),
+                p.amount.single.calc(reduced_agi, right=True),
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/wv/tax/income/credits/liftc/wv_low_income_family_tax_credit.py:35]: dont think we need the right true here

File: policyengine_us/variables/gov/states/wv/tax/income/credits/liftc/wv_low_income_family_tax_credit.py [hunk 2]
```diff
@@ -31,16 +32,17 @@ def formula(tax_unit, period, parameters):
...
+                p.amount.head_of_household.calc(reduced_agi, right=True),
+                p.amount.widow.calc(reduced_agi, right=True),
             ],
         )
 
         tax_before_credits = tax_unit(
             "wv_income_tax_before_non_refundable_credits", period
         )
-
+        print(credit_percentage)
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/wv/tax/income/credits/liftc/wv_low_income_family_tax_credit.py:46]: delete the print statements

File: policyengine_us/tests/policy/baseline/gov/states/wv/tax/income/credits/liftc/wv_low_income_family_tax_credit.yaml [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    wv_low_income_family_tax_credit_fpg: 27_750
+    state_code: WV
+    filing_status: SINGLE
+    wv_income_tax_before_non_refundable_credits: 1_000
+    state_group: CONTIGUOUS_US
+  output:
+   wv_low_income_family_tax_credit: 0
+
+
+#  brackets interval: wv_modified_agi-tax_unit_fpg*percent
```
Comment (PavelMakarchuk) [inline policyengine_us/tests/policy/baseline/gov/states/wv/tax/income/credits/liftc/wv_low_income_family_tax_credit.yaml:14]: delete

File: policyengine_us/parameters/gov/states/wv/tax/income/credits/liftc/amount/head_of_household.yaml
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/wv/tax/income/credits/liftc/amount/head_of_household.yaml:1]: fix the yaml files with the "-" in front of the amount

**Review by PavelMakarchuk — APPROVED** [review]

**Review by MaxGhenis — APPROVED** [review]

File: changelog_entry.yaml
Comment (PavelMakarchuk) [inline changelog_entry.yaml:1]: could you run a `make format` and then a `git pull upstream master` - let me know if this fixes the PR
